lib/boombox.ex

```diff
@@ -140,6 +148,12 @@ defmodule Boombox do
       {:webrtc, uri} when is_binary(uri) ->
         value
 
+      {:whip, opts} when is_list(opts) ->
+        if Keyword.keyword?(opts) do
+          opts = parse_whip_opts(opts)
+          {:webrtc, {:whip, opts}}
+        end
```

Comment:
I guess we should raise if `Keyword.keyword?/1` returns false

Reply:
we do here https://github.com/membraneframework/boombox/pull/49/files#diff-dc09e5f55160df34c7f0294fda9202894bd0e4d294dfcc14d5a1cece6a09f9a8R173